napari/_qt/range_slider/range_slider.py

```diff
@@ -113,28 +114,28 @@ def mouseMoveEvent(self, event):
             size = self.rangeSliderSize()
             pos = self.getPos(event)
             if self.moving == "min":
```

Comment:
Does `yapf` not force `'` instead of `"`. I prefer `'`, but is seems strange to me that the auto formatter allows both

Reply:
`yapf` does not force any by default